jellyfish/_notifier.py

```diff
@@ -104,26 +114,28 @@ async def wait_ready(self) -> True:
         await self._ready_event.wait()
 
     async def _authenticate(self):
-        msg = ServerMessage(auth_request=ServerMessageAuthRequest(token=self._server_api_token))
+        msg = ServerMessage(
+            auth_request=ServerMessageAuthRequest(token=self._server_api_token)
+        )
         await self._websocket.send(bytes(msg))
 
         try:
             message = await self._websocket.recv()
         except ConnectionClosed as exception:
-            if 'invalid token' in str(exception):
```

Comment:
Is there a way to configure formatting of string literals? If its easy to setup I would personally prefer single quotes around such strings. I think its more 'pythonic', also I've got this CKAN coding standard to back up my opinion: https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html

Reply:
Update: I read the black docs regarding this matter and I withdraw the charges. Lets stick to black's double quotes for every string.